### ISSUE TITLE
Please support PulseAudio or indicate some workaround how to use it

### ISSUE THREAD

**Issue by ghost** [dj0abr/SSB_HighSpeed_Modem#5]
Now that transmissions have started I have installed the software on my Linux system.
Unfortunately it can only open physical sound interfaces...
My system uses PulseAudio as a "patchboard" for audio between applications, I can e.g. use FLdigi or QSSTV as a demodulator for audio produced by the SDR receiver program GQRX.
But the SSB_HighSpeed_Modem does not receive audio from this.

**Comment by df7cb** [comment]
#4 should fix this I think. Unfortunately I haven't got any response on it yet.

**Comment by ghost** [comment]
Yes, after entering this issue I saw that there are pull requests but they are not being merged.
Also there is both a personal (this) github entry and one for Amsat-DL: https://github.com/amsat-dl/QO-100-Modem

Hopefully now that the modem is becoming active the author will merge the fixes and the github page will be made 1 or the same, so we all know where we can get the best version.

**Comment by dj0abr** [comment]
thanks to all for the feedback.
I have been busy with the QO100 MM-Beacon, but this is done now, so will now start working on releasing a new version addressing all your request.
